docs/STAND-ALONE_MODE.md

```diff
@@ -0,0 +1,39 @@
+# Stand-alone mode
+
+Stand-alone mode enables plugin launching on different machine than Snap daemon (`snapteld`).
+This feature works for plugins written using one of our snap-plugin-libs ([snap-plugin-lib-go](https://github.com/intelsdi-x/snap-plugin-lib-go),
+[snap-plugin-lib-py](https://github.com/intelsdi-x/snap-plugin-lib-py), [snap-plugin-lib-cpp](https://github.com/intelsdi-x/snap-plugin-lib-cpp)).
+
+## Running a plugin in stand-alone mode
+To run a plugin in stand-alone mode, you must start it with the `--stand-alone` flag, e.g.:
+```
+$ ./snap-plugin-collector-psutil --stand-alone
+```
+
+A plugin running in stand-alone mode creates a HTTP server for communication with the Snap framework.
+By default the plugin listens on port `8182`.
+
+To specify a different listening port, use the `--stand-alone-port` flag, e.g.:
+```
+$ ./snap-plugin-collector-psutil --stand-alone --stand-alone-port 8183
+```
+##  Loading a plugin
+When loading a plugin in stand-alone mode into `snapteld` you must provide a URL to indicate which
+machine the plugin is running (IP address/hostname with port number), e.g.:
```

Comment:
Consider rephrasing 
`When loading a plugin in stand-alone mode into `snapteld` you must provide a URL to indicate which machine the plugin is running (IP address/hostname with port number), e.g.:`
to

`To load a plugin in stand-alone mode, provide a URL to indicate to the machine on which the plugin is running (IP address/hostname with port number), e.g.:`